### PR TITLE
feat: Reply and forward icon added

### PR DESCRIPTION
**Pull request by Ambrdctr**

### CODE REVIEW

File: Mail/Components/ThreadCell/ThreadCellAnsweredView.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            return MailResourcesAsset.emailForwardFilled.swiftUIImage
+        }
+        return nil
+    }
+
+    var body: some View {
+        image?
+            .frame(width: 16)
+            .foregroundStyle(MailResourcesAsset.textSecondaryColor.swiftUIColor)
+            .padding(.trailing, 4)
```
Comment (valentinperignon) [inline Mail/Components/ThreadCell/ThreadCellAnsweredView.swift:39]: We can use a UIPadding here

File: Mail/Components/ThreadCell/ThreadCellAnsweredView.swift [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        if answered {
+            return MailResourcesAsset.emailReplyFilled.swiftUIImage
+        } else if forwarded {
+            return MailResourcesAsset.emailForwardFilled.swiftUIImage
+        }
+        return nil
+    }
+
+    var body: some View {
+        image?
```
Comment (valentinperignon) [inline Mail/Components/ThreadCell/ThreadCellAnsweredView.swift:36]: If you apply a .frame(), the image should be .resizable()

File: Mail/Components/ThreadCell/ThreadCellAnsweredView.swift [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            .frame(width: 16)
+            .foregroundStyle(MailResourcesAsset.textSecondaryColor.swiftUIColor)
```
Comment (valentinperignon) [inline Mail/Components/ThreadCell/ThreadCellAnsweredView.swift:38]: We have a custom foregroundStyle modifier so we don't need to type swiftUIColor each time
```suggestion
            .foregroundStyle(MailResourcesAsset.textSecondaryColor)
```

File: Mail/Components/ThreadCell/ThreadCellBodyView.swift [hunk 4]
```diff
@@ -17,6 +17,7 @@
  */
 
 import MailCore
+import MailResources
```
Comment (valentinperignon) [inline Mail/Components/ThreadCell/ThreadCellBodyView.swift:20]: Is MailResources required in this file?

File: MailResources/Assets.xcassets/email-reply-filled.imageset/email-reply-filled.svg
Comment (valentinperignon) [inline MailResources/Assets.xcassets/email-reply-filled.imageset/email-reply-filled.svg:1]: Is it possible to have a 16x16 icon?

File: MailResources/Assets.xcassets/email-forward-filled.imageset/email-forward-filled.svg
Comment (valentinperignon) [inline MailResources/Assets.xcassets/email-forward-filled.imageset/email-forward-filled.svg:1]: Is it possible to have a 16x16 icon?

File: Mail/Components/ThreadCell/ThreadCellAnsweredView.swift [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        return nil
+    }
+
+    var body: some View {
+        image?
+            .resizable()
+            .scaledToFit()
+            .frame(width: 14)
+            .foregroundStyle(MailResourcesAsset.textSecondaryColor)
+            .padding(.trailing, UIPadding.verySmall)
```
Comment (valentinperignon) [inline Mail/Components/ThreadCell/ThreadCellAnsweredView.swift:42]: You might be able to do this:
```suggestion
            .padding(.trailing, value: .verySmall)
```

File: Mail/Components/ThreadCell/ThreadCellAnsweredView.swift [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        if answered {
+            return MailResourcesAsset.emailReplyFilled.swiftUIImage
+        } else if forwarded {
+            return MailResourcesAsset.emailForwardFilled.swiftUIImage
+        }
+        return nil
+    }
+
+    var body: some View {
+        image?
```
Comment (valentinperignon) [inline Mail/Components/ThreadCell/ThreadCellAnsweredView.swift:37]: @TommyDL-Infomaniak created two accessibility labels:
- contentDescriptionIconForward
- contentDescriptionIconReply
Can we integrate them?

File: Mail/Components/ThreadCell/ThreadCellAnsweredView.swift [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+            .resizable()
+            .scaledToFit()
+            .frame(width: 14)
```
[the remark this answers is not in the document]
Comment (Ambrdctr) [reply Mail/Components/ThreadCell/ThreadCellAnsweredView.swift:40]: Updated to 16.
The scaledToFit is still necessary.

**Review by valentinperignon — APPROVED** [review]

**Pull request merged**